Makefile

```diff
@@ -230,10 +230,10 @@ plugin-%:
 	@echo "Starting dev environment for $${$(@)} input plugin..."
 	@docker-compose -f plugins/inputs/$${$(@)}/dev/docker-compose.yml up
 
-.PHONY: ci-1.17
-ci-1.17:
-	docker build -t quay.io/influxdb/telegraf-ci:1.17.7 - < scripts/ci-1.17.docker
-	docker push quay.io/influxdb/telegraf-ci:1.17.7
+.PHONY: ci-1.18
```

Comment:
if we are only maintaining one version, any reason to note rename this to something version independent? e.g. `ci`

Reply:
Good point! Sounds like a good change, two less things to update when going to a new version. I've pushed a change.